gems/aws-sdk-s3/spec/presigner_spec.rb

```diff
@@ -43,6 +43,20 @@ module S3
 
       describe "#presigned_url" do
 
+        it 'will be tracked as an api request' do
+          stub_client = Aws::S3::Client.new(stub_responses: true)
+          presigner = Presigner.new(client: stub_client)
+          presigner.presigned_url(:get_object, bucket: 'bkt', key: 'k')
+          expect(stub_client.api_requests.size).to eq(1)
+        end
+
+        it 'can be excluded from being tracked as an api request' do
+          stub_client = Aws::S3::Client.new(stub_responses: true, exclude_presign: true)
+          presigner = Presigner.new(client: stub_client)
+          presigner.presigned_url(:get_object, bucket: 'bkt', key: 'k')
+          expect(stub_client.api_requests.size).to be_zero
```

Comment:
I actually imagined this as an option you could pass in to the `api_requests` method itself. Should be easy to replace the attr_reader with a method that takes an optional param to do this. That way, you don't need to decide at client construction time.